src/domains/wallet-common/env/weu-uat/terraform.tfvars

```diff
@@ -24,8 +24,12 @@ log_analytics_workspace_resource_group_name = "pagopa-u-monitor-rg"
 
 ingress_load_balancer_ip = "10.1.100.250"
 
+## DNS
+
 external_domain          = "pagopa.it"
 dns_zone_internal_prefix = "internal.uat.platform"
+dns_zone_prefix          = "uat.wallet"
```

Comment:
ditto

Reply:
Fixed in 9f2a4bf6792e5fa01f309595f1e445a690468fdd